Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM gcr.io/distroless/cc-debian12:nonroot
+FROM gcr.io/distroless/cc-debian12:debug-nonroot
```

Comment:
why this change?

Reply:
Thanks, that is a leftover from debugging and shouldn't be there. Reverted now.